src/pkg/utils/component_report.go

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package utils provides generic helper functions.
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/defenseunicorns/zarf/src/config/lang"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	"github.com/defenseunicorns/zarf/src/types"
+	"github.com/openvex/go-vex/pkg/vex"
+)
+
+// VexType is the type of report that is a VEX document
+const VexType = "vex"
+
+// ZarfReport Extends src/types.ZarfComponentReport to add methods
+type ZarfReport struct {
+	ZarfComponentReport types.ZarfComponentReport
+}
+
+// ValidateReportSource validates vex type reports
+func (report *ZarfReport) ValidateVexReport(filePath string) error {
```

Comment:
lint: `comment on exported method ZarfReport.ValidateVexReport should be of the form "ValidateVexReport ..."`

Reply:
fixed thanks